apps/nextjs/src/components/ExportsDialogs/useExistenceCheck.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from "react";
+
+import * as Sentry from "@sentry/react";
+
+export const useExistenceCheck = <T>({
```

Comment:
Potentially name this with a more explicit definition of what the existence check is for?

Reply:
Yep sure. It's scoped within the exports folder, but agree it could have a better name

Reply:
Done